build.gradle

```diff
@@ -79,35 +76,35 @@ repositories {
 
 wrapper {
     distributionType = Wrapper.DistributionType.ALL
-    gradleVersion = "5.3.1"
+    gradleVersion = "6.2.2"
 }
 
 
-def awsSdkVersion = "1.11.542"
-def jupiterVersion = "5.4.2"
+def awsSdkVersion = "1.11.739"
+def jupiterVersion = "5.6.0"
 
 dependencies {
-    implementation ("com.amazonaws:aws-java-sdk-api-gateway:${awsSdkVersion}")
-    implementation ("com.amazonaws:aws-java-sdk-cloudformation:${awsSdkVersion}")
-    implementation ("com.amazonaws:aws-java-sdk-codepipeline:${awsSdkVersion}")
-    implementation ("com.amazonaws:aws-java-sdk-route53:${awsSdkVersion}")
-    implementation ("com.amazonaws:aws-java-sdk-secretsmanager:${awsSdkVersion}")
+    implementation("com.amazonaws:aws-java-sdk-api-gateway:${awsSdkVersion}")
+    implementation("com.amazonaws:aws-java-sdk-cloudformation:${awsSdkVersion}")
+    implementation("com.amazonaws:aws-java-sdk-codepipeline:${awsSdkVersion}")
+    implementation("com.amazonaws:aws-java-sdk-route53:${awsSdkVersion}")
+    implementation("com.amazonaws:aws-java-sdk-secretsmanager:${awsSdkVersion}")
 
     // For reading and deleting logs (application functionality)
-    implementation ("com.amazonaws:aws-java-sdk-logs:${awsSdkVersion}")
+    implementation("com.amazonaws:aws-java-sdk-logs:${awsSdkVersion}")
 
     //this should be the same version as the one used by amazon
-    implementation ("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.6.0")
+    implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.10.3")
 
     // useful java libraries
-    implementation ("com.google.guava:guava:27.1-jre")
+    implementation("com.google.guava:guava:27.1-jre")
 
     implementation('io.symphonia:lambda-logging:1.0.3')
```

Comment:
We should probably just use System.out for logging

Reply:
made an issue as a reminder